### PR TITLE
DOLLOSA: survey-form project, chess-flexbox, and chess-grid

### PR DESCRIPTION
**Pull request by michael-dollosa**
- survey form project submission

### CODE REVIEW

**Comment by michael-dollosa** [comment]
Pushed chess-flexbox project
-added .gitignore for node_modules

**Comment by michael-dollosa** [comment]
Pushed chess-grid project
-added .gitignore for node_modules

**Comment by pau-riosa** [comment]
> Pushed chess-grid project
> -added .gitignore for node_modules

what package did you install and use?

File: dollosa-michael/chess-flex/css/style.output.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  width: 10vmin;
+  height: 10vmin;
+  box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.2);
+  overflow: hidden;
+  position: relative;
+  font-size: 6rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center; }
+  @media only screen and (max-width: 700px) {
```
Comment (pau-riosa) [inline dollosa-michael/chess-flex/css/style.output.css:53]: good job on applying responsive in your work. keep it up.

Comment (michael-dollosa) [reply dollosa-michael/chess-flex/css/style.output.css:53]: "what package did you install and use?"
- installed node-sass, npm-run-all, and concat lang sir

**Pull request merged**